docs/contributing/doc_guide.rst

```diff
@@ -1184,40 +1184,37 @@ It may also be necessary to install the following software:
 
 * `graphviz <https://graphviz.org/download>`__
 * `pandoc <https://pandoc.org/installing.html>`__
-* make_ (not necessary for building the documentation with |tox| or
+* make_ (not necessary for building the documentation with |nox| or
   sphinx_build)
 
 Building documentation
 ----------------------
 
-PlasmaPy's documentation can be built using |tox|, make_, or
-sphinx-build_. We recommend starting with |tox|.
+PlasmaPy's documentation can be built using |nox|, make_, or
+sphinx-build_. We recommend starting with |nox|.
 
 .. tabs::
 
-   .. group-tab:: tox
+   .. group-tab:: nox
 
-      We can use |tox| to build the documentation locally by running:
+      We can use |nox| to build the documentation locally by running:
 
       .. code-block:: bash
 
-         tox -e build_docs
+         nox -s docs
```

Comment:
tox -e replaced with nox -s, assuming they are the correct letters but just checking

Reply:
Yep, this is important to double check!   nox primarily uses `-s` because it is short for "session", while tox uses `-e` primarily because it is short for "environment".

(The `-e` also works for nox, but I figured it'd be better to be consistent with the nox docs.)